### ISSUE TITLE
Support for ZLE vi-mode

### ISSUE THREAD

**Issue by nimish** [marlonrichert/zsh-autocomplete#192]
## What do you want?
zle vi-mode support
## Why do you want this?
For people who use vi-mode

## Explain in detail how you would expect this to work
I expect that running `bindkey -v` will not break keybindings for zsh-autocomplete and/or there's an integration like there appears to be with emacs-mode

**Comment by marlonrichert** [comment]
@nimish I created pull request #193 with key bindings for `vi` mode, but since I don't use `vi` mode myself, I am not 100% that they make sense to `vi` users. Please do
```zsh
% git fetch origin
% git checkout -b vi-mode origin/vi-mode
```
to try them out for yourself and let me know what you think.

**Comment by nimish** [comment]
Awesome. Works well enough for me!

Only confusion was that I have `history-substring-search-up/down` mapped to Up/Down, so had to repeat that for vicmd mode -- nbd (maybe a feature req to allow config of those?)

**Comment by marlonrichert** [comment]
I think you already found how to configure this: [Just use `bindkey`](https://github.com/marlonrichert/zsh-autocomplete#change-other-key-bindings). 🙂

**Comment by nimish** [comment]
Cool, I think this looks good to me, and it can always be customized later

**Comment by marlonrichert** [comment]
Merged to `main` in d4580dedcb0fec7e1dbce1e09086efa8647cc7d1.
